gutenberg.php

```diff
@@ -210,6 +210,8 @@ function gutenberg_init( $return, $post ) {
 		return false;
 	}
 
+	get_current_screen()->is_block_editor( true );
```

Comment:
I saw a comment on the description, but still having hard time understanding this :)

Reply:
> I saw a comment on the description, but still having hard time understanding this :)

It mimics similar to what's done here:

https://github.com/WordPress/wordpress-develop/blob/8b4330bb6ea196005b5f0f84070b64c139f39b37/src/wp-admin/edit-form-blocks.php#L26-L28

It's important because there are conditions in core's handling of meta boxes which key off this value:

- https://github.com/WordPress/wordpress-develop/blob/8b4330bb6ea196005b5f0f84070b64c139f39b37/src/wp-admin/includes/template.php#L1242
- https://github.com/WordPress/wordpress-develop/blob/8b4330bb6ea196005b5f0f84070b64c139f39b37/src/wp-admin/includes/template.php#L1252
- https://github.com/WordPress/wordpress-develop/blob/8b4330bb6ea196005b5f0f84070b64c139f39b37/src/wp-admin/includes/template.php#L1265
- https://github.com/WordPress/wordpress-develop/blob/8b4330bb6ea196005b5f0f84070b64c139f39b37/src/wp-admin/includes/template.php#L1290

Without this, the the value of `is_block_editor` is _false_ on the Gutenberg screen, which is clearly problematic irrespective of the specific meta box integration. For example, it forces plugin authors to do funny things to check whether the current screen is the block editor, regardless of whether that's the one in core or the one in the Gutenberg plugin.

https://github.com/aduth/g-debugger/blob/522d88a56f7e389f741edc48220a9750d78b80fd/g-debugger.php#L13-L32

Reply:
I added a clarifying comment in the rebased 4f62c93.